sdks/go/alpha.go

```diff
@@ -84,3 +83,96 @@ func (a *Alpha) GetConnectedPlayers() ([]string, error) {
 	list, err := a.client.GetConnectedPlayers(context.Background(), &alpha.Empty{})
 	return list.GetList(), errors.Wrap(err, "could not list connected players")
 }
+
+// GetCounter returns a copy of a Counter, given the Counter's key (name).
+// Will error if the key was not predefined in the GameServer resource on creation.
+func (a *Alpha) GetCounter(key string) (alpha.Counter, error) {
```

Comment:
Do we want or not want `GetCounter`? It wasn't a part of the #2716 specification, however it seems like a standard method to have.

Reply:
🤔 personally I don't _think_ we need it?

My thought - let's remove it, we can always  addit later if feedback requires, rather than have it now and have to maintain it going forward?